Backpack/StarRating/Classes/BPKHotelStarRating.swift

```diff
@@ -0,0 +1,95 @@
+//
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright © 2023 Skyscanner Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import UIKit
+
+public class BPKHotelStarRating: UIView {
```

Comment:
Would it make sense to make this a UIStackView subview? @frugoman 
That way we remove a layer of nesting?

Reply:
It make sense. That also makes the component simpler :)

Reply:
Agree on this one! good catch!